bazel/thirdparty/libpciaccess.BUILD

```diff
@@ -2,6 +2,18 @@
 # This build is a translation of the libpciaccess meson build:
 # https://gitlab.freedesktop.org/xorg/lib/libpciaccess/-/blob/2ec2576cabefef1eaa5dd9307c97de2e887fc347/meson.build
 #
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+# Selects with AND'd conditions require a standalone helper.
+# https://bazel.build/docs/configurable-attributes#and-chaining
+selects.config_setting_group(
+    name = "linux_x86_64",
+    match_all=[
+        "@platforms//cpu:x86_64",
+        "@platforms//os:linux",
+    ],
+)
```

Comment:
Redpanda only supports linux, so we've been lazy and just using the cpu constraint but it's probably better to be more explicit here so thanks :)

Reply:
TBF it's weirdly painful to setup a constraint on cpu + os for some reason... I mostly did this as education 😓